### DISCUSSION TITLE
Renderização condicional de componentes

### DISCUSSION THREAD

**Discussion by Franciscof11** [Flutterando/forum#112] [Perguntas e Respostas]
Olá galera, sou novo aqui na comunidade, meu primeiro post, mas vamo lá .
Estou com um projeto em flutter, e tenho que implementar uma tela dinâmica de boletim, onde as regras de nota influenciam na cor do widget que irá aparecer.
Sendo nota < 7 ( Vermelho ) , nota > 7 ( Verde ) .
Como eu posso implementar essa renderização com condicional no flutter? 

Essa é a tela : 
![image](https://user-images.githubusercontent.com/65028917/212473075-55a49e57-27a6-4ce1-9d7d-f1f79ee8b38f.png)

Obrigado pela atenção :)

**Comment by eduardoflorence** [comment]
Olá @Franciscof11, você pode fazer assim:

```dart
Container(
  padding: const EdgeInsets.all(8),
  width: 50,
  decoration: BoxDecoration(
    color: nota >= 7.0 ? Colors.green : Colors.red,
    borderRadius: BorderRadius.circular(8)
  ),
  child: Text(
    nota.toStringAsFixed(1),
    style: const TextStyle(color: Colors.white),
    textAlign: TextAlign.end,
  )
)
```

Ou seja, para renderização condicional você utiliza o operador ternário `condição ? verdadeiro : falso`
Veja: https://dart.dev/guides/language/language-tour#conditional-expressions

**Comment by Franciscof11** [comment]
Vlw mano!!
